modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Answer.es.js

```diff
@@ -313,9 +314,10 @@ export default withRouter(
 						<Comments
 							comments={comments}
 							commentsChange={_commentsChange}
-							companyMx={comments.companyMx}
+							companyName={context.companyName}
```

Comment:
Do we really need to pass companyName here, since the Comments component has access to context? 

```suggestion
							companyName={context.companyName}
```